tests/unit/src/test/resources/semanticTokens/example/MacroAnnotation.scala

```diff
@@ -0,0 +1,19 @@
+<<package>>/*keyword*/ <<example>>/*namespace*/
+
+<<import>>/*keyword*/ io.circe.derivation.annotations.<<JsonCodec>>/*variable,readonly*/
```

Comment:
We should probably add the classpath to the tests, but I can take a look at that later.